lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm

```diff
@@ -11,6 +11,7 @@ no indirect;
 
 use curry;
 
+use DataDog::DogStatsd::Helper qw(stats_inc);
```

Comment:
We can't use this directly here, otherwise we have to update the `cpanfile` dependency to require this module.  Instead, a callback should be supplied so that users of this module can introduce their own metrics.